src/app/pages/applications/catalog/catalog.component.html

```diff
@@ -3,29 +3,28 @@
         <div class="content-wrapper" fxLayout="row">
           <div class="logo-container">
             <div class="logo" style="background-image: url('{{ item.icon_url}}');"></div>
+            <div class="badge-area">
+                <div class="partner-badge capitalized-text">
+                    {{ item.catalog.label }}
+                </div>                    
+                <div class="train-badge capitalized-text">
+                    {{ item.catalog.train }}
+                </div>                    
+            </div>
           </div>
             <div class="content" fxLayout="column" fxLayoutAlign="center center">
-                <div class="catalog-title">
+                <div class="catalog-title" (click)="showSummaryDialog(item.name)">
                     <h3 textLimiter threshold="10">{{ item.name }}</h3>
```

Comment:
This should fix most longer-named Apps being cut-off too soon:
```suggestion
                <div class="catalog-title" (click)="showSummaryDialog(item.name)">
                    <h3 textLimiter threshold="12">{{ item.name }}</h3>
```

I've manually tested this, with multiple resolutions...
While "Transmission" (12 characters wide) now displays correctly, it doesn't  cause any unexpected sideeffects on other App names